### PR TITLE
Feature/display collections

### PR DESCRIPTION
**Pull request by yuchenglin**
This pull request makes the following changes:
- Display collection names in the popup windows of map view.

Test these changes by:
- Update both /platform and /platform-client according to this pull request. There would be collection names in each popup windows.

Refs ushahidi/platform#1070

Ping @ushahidi/platform

---
This change is [Reviewable](https://reviewable.io/reviews/ushahidi/platform-client/179)

### CODE REVIEW

File: app/common/services/maps.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ function (
                 '<strong><a href="/posts/' + feature.properties.id + '">' +
                 title +
                 '</a></strong>' +
-                '<p>' + description + '</p>'
+                '<p>' + description + '</p>' +
+                '<p><b>Collection: </b>' + feature.properties.set_name + '</p>'
```
Comment (rjmackay) [inline app/common/services/maps.js]: isn't `set_name` going to be an array?

Comment (yuchenglin) [inline app/common/services/maps.js]: It's an array, and I would rename it as `collections`

Comment (rjmackay) [inline app/common/services/maps.js]: Could you use `collections.join(', ')` so they're joined with a space after the comma too?

Comment (yuchenglin) [inline app/common/services/maps.js]: Sure!

File: app/common/services/maps.js [hunk 2]
```diff
@@ -53,12 +53,17 @@ function (
     var geojsonLayerOptions = {
         onEachFeature: function (feature, layer) {
             var description = feature.properties.description || '',
-                title = feature.properties.title || feature.properties.id;
+                title = feature.properties.title || feature.properties.id,
+                collections = [];
+            for (var i in feature.properties.collections) {
+                collections.push(feature.properties.collections[i].name);
```
Comment (rjmackay) [inline app/common/services/maps.js]: You could do this easily with underscore `collections = _.pluck(feature.properties.collections, 'name')`

Comment (yuchenglin) [inline app/common/services/maps.js]: Okay!

**Comment by rjmackay** [comment]
@yuchenglin sorry about the delay on this. We're currently in the process of a big UI shift - that'll be done next week and I can come back to this.
At the moment we don't have collections visible in a posts preview or full view.. so I'm not sure if this should make it into core UI (I'll check in with our design team). However we should make the UI changes so you can do this on your own deployment if needed.

**Comment by yuchenglin** [comment]
Thanks for your reply. I'm looking forward to the big UI shift.

**Comment by rjmackay** [comment]
@yuchenglin I don't think we're likely to integrate this with our UI at this stage. You're obviously welcome to keep it on your fork though. Happy to try to get the API changes into core though.

**Pull request closed without merging**